api/authenticators/onelogin_duo/authenticator_test.go

```diff
@@ -1,28 +0,0 @@
-package oneloginduo
```

Comment:
This test was removed because it can be done at compile time with a type assertion